### PR TITLE
Feature/title label

### PR DESCRIPTION
**Pull request by olmesm**

### CODE REVIEW

**Comment by rajeshwarpatlolla** [comment]
Instead of title it was intentionally made changes in the code to show the selected date.

**Comment by olmesm** [comment]
That's still the default, but now optionally allows for a title to be added?

File: src/ionic-datepicker-popup.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<div class="selected_date_full">{{selctedDateEpoch | date : mainObj.dateFormat}}</div>
+<div class="selected_date_full">{{mainObj.titleLabel || selctedDateEpoch | date : mainObj.dateFormat}}</div>
```
Comment (olmesm) [inline src/ionic-datepicker-popup.html]: @rajeshwarpatlolla

**Comment by rajeshwarpatlolla** [comment]
When you open the datepicker it will show the input date which is passed to the component. Once you select some other date, then it will show the selected date on the header. So we don't have a scenario where we can show the title instead of date.

**Comment by olmesm** [comment]
`{{mainObj.titleLabel || selctedDateEpoch | date : mainObj.dateFormat}}`

The changes I've requested still do **exactly** what you are saying.

But if you set `titleLabel` it then allows you to specify a title - an additional feature.

This is an old feature you had and something that could still be needed by others as well as myself.

**Comment by rajeshwarpatlolla** [comment]
Thanks for your PR.

**Comment by domharrington** [comment]
Can you publish a new version to npm?

**Pull request merged**